lib/client/dio.dart

```diff
@@ -23,7 +23,7 @@ Dio httpClient(
 
   final options = BaseOptions(
     baseUrl: 'https://api.weatherapi.com/v1',
-    queryParameters: {'key': '<your-api-key>'}, // DO NOT commit this key
+    queryParameters: {'key': '065505914dc344ad95c121519231307'}, // DO NOT commit this key
```

Comment:
Dang, you've committed a key here (given it's a valid API key). You now have to:

1. delete this key or invalidate it via weatherapi.com
2. delete this change
3. force a commit in which this change never happened OR simply start over again.

Reply:
I also checked before doing the PR. Sorry, I think it's time to switch to an .env file.

Reply:
Yes absolutely it's in my TODOs for this project. I'm sorry I didn't do it yet.